samples/maps/demo/basic-map/demo.details

```diff
@@ -0,0 +1,12 @@
+---
+name: World High Resolution
+authors:
+  - Torstein Hønsi
+js_wrap: b
+tags:
+  - Highcharts Maps demo
+categories:
+  - General:
+      priority: 1
+use_png_thumbnail: true
```

Comment:
Are we sure? @goransle

Reply:
SVG works, so this property can be removed.
![thumbnail](https://github.com/highcharts/highcharts/assets/7478772/0bbf33e0-41fb-48b5-9526-5e77ba3f8aaa)